src/utils/py/daos_api.py

```diff
@@ -1629,6 +1629,82 @@ def get_attr(self, attr_names, coh=None, cb_func=None):
 
         return results
 
+class DaosSnapshot(object):
```

Comment:
coordinate with stephen on adding this to his new readme.md file.

Reply:
Will do.